gallery/src/components/NxGlobalSidebar/NxGlobalSidebarPage.tsx

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import CodeExample from '../../CodeExample';
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import { NxP, NxCode, NxTable, NxTile, NxH2, NxH3 } from '@sonatype/react-shared-components';
+
+import './NxGlobalSidebarPage.scss';
+
+const NxGlobalSidebarExample = require('./NxGlobalSidebarExample.tsx?raw');
+
+export default function NxGlobalSidebarPage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <NxP>
+          <NxCode>NxGlobalSidebar</NxCode> is a collapsible page level sidebar. It has two main sections:
+        </NxP>
+        <ul className="nx-list nx-list--bulleted">
+          <li className="nx-list__item">A header which contains branding as well as the open/close toggle</li>
+          <li className="nx-list__item">
+            A navigation link section which display an icon plus text in the open state and an icon in the closed
+            state
+          </li>
+        </ul>
+        <NxP>
+          Because <NxCode>NxGlobalSidebar</NxCode> contains product branding it should not be used in conjunction
+          with <NxCode>NxPageHeader</NxCode> or <NxCode>NxNexusPageHeader</NxCode>. Instead it should be paired with
+          <NxCode>NxSystemHeader</NxCode> (not yet added to the gallery).
+        </NxP>
+        <NxP>
+          Additional custom content may be added below the navigation links if desired, care should be taken that such
+          content can adapt to both the open and closed states.
+        </NxP>
+        <NxTile.Subsection>
+          <NxTile.SubsectionHeader>
+            <h3 className="nx-h3"><NxCode>NxGlobalSidebar</NxCode> Props</h3>
+          </NxTile.SubsectionHeader>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>isOpen</NxTable.Cell>
+                <NxTable.Cell>Boolean</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>This value determines whether the side starts in an open or closed state.</NxTable.Cell>
```

Comment:
It's not whether it starts open or closed, it's whether it is open or closed.  This being the stateless version of the component.

Reply:
> It's not whether it starts open or closed, it's whether it is open or closed. This being the stateless version of the component.

https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/8302c5865720b50cc8af378bb27c9e3882751783